### PR TITLE
Integrate Viralize on Google AMP AdNetwork

### PR DESCRIPTION
**Pull request by canemacchina**
Integration of Viralize player into AMP ad network list.

### CODE REVIEW

**Comment by canemacchina** [comment]
We signed the CLA.

File: examples/ads.amp.html [hunk 1]
```diff
@@ -1426,6 +1427,20 @@ <h2>ValueCommerce</h2>
       data-pid="884466614">
   </amp-ad>
 
+  <h2>Viralize</h2>
+  <amp-ad width="300" height="169"
+      type="viralize"
+      layout="responsive"
+      data-zid="AACX6WjIMkl-yoGV"
+      data-extra='{
```
Comment (zhouyx) [inline examples/ads.amp.html]: nit: please fix indentation here.

File: ads/viralize.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+function urlEncode(dict) {
```
[the remark this answers is not in the document]
Comment (canemacchina) [reply ads/viralize.js]: Ok, `addParamsToUrl` is what I need, so I will use this function instead 👍

**Comment by canemacchina** [comment]
Ok I've make the changes to the code as you requested! 
Just one question: what we are missing to pass the CLA requirement? I don't understand sorry!

File: build-system/conformance-config.textproto [hunk 3]
```diff
@@ -187,6 +187,7 @@ requirement: {
   whitelist: 'ads/kargo.js'
   whitelist: 'ads/mads.js'
   whitelist: 'ads/google/imaVideo.js'
+  whitelist: 'ads/viralize.js'
```
Comment (zhouyx) [inline build-system/conformance-config.textproto]: pleaser revert this file

**Comment by canemacchina** [comment]
I'm sorry @zhouyx, I think I've discovered the problem with the CLA, it was a misconfiguration of my github account: my primary github email address wasn't added to the google group.
I have had to change my primary email an sign again my commits. Now I think everything should be fine.
Could you please check? thanks

**Comment by lannka** [comment]
@erwinmombay could you help to merge?

**Pull request merged**